nominations/templates/application_status.html

```diff
@@ -32,20 +32,19 @@ <h1 class="mb0 mt0">{{application}}</h1>
             <li class="list-group-item"><span class="glyphicon glyphicon-check"></span> 1. Start an Application</li>
             <li class="list-group-item"><a href="/groups/nominations/nomination?id={{application.id}}"><span class="glyphicon {% if application.nomination.status == 'complete' %}glyphicon-check{% else %}glyphicon-unchecked{% endif %}"></span> 2. Complete the group nomination form</a></li>
             <li class="list-group-item"><a href="/groups/nominations/questionnaire?id={{application.id}}"><span class="glyphicon {% if application.questionnaire.status == 'complete' %}glyphicon-check{% else %}glyphicon-unchecked{% endif %}"></span> 3. Have your candidate fill out the candidate questionaire</a></li>
-            
-            {% if application.status == 'incomplete' %}
-              <li class="list-group-item"><a href="/groups/nominations/submit?id={{application.id}}"><span class="glyphicon {% if application.status == 'submitted' %}glyphicon-check{% else %}glyphicon-unchecked{% endif %}"></span> 4. Submit your nomination</a></li>
-            {% else %}  
+
+            {% if application.is_editable %}
+              <li class="list-group-item"><a href="/groups/nominations/submit?id={{application.id}}"><span class="glyphicon glyphicon-unchecked"></span> 4. Submit your nomination</a></li>
+            {% else %}
               <li class="list-group-item"><span class="glyphicon glyphicon-check"></span> 4. Submit your nomination</li>
```

Comment:
Suggest to make this more DRY - don't repeat `4. Submit your nomination` twice in the code.

Reply:
👍